Continuum/Anaconda3.munki.recipe

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>Description</key>
+	<string>Downloads the current release version of Anaconda and imports it into Munki.</string>
+	<key>Identifier</key>
+	<string>com.github.its-unibas.munki.Anaconda3</string>
+	<key>Input</key>
+	<dict>
+		<key>NAME</key>
+		<string>Anaconda</string>
+		<key>PYTHON_MAJOR_VERSION</key>
+		<string>3</string>
+		<key>INSTALLER_TYPE</key>
+		<string>pkg</string>
+		<key>MUNKI_CATALOGS</key>
+		<string>testing</string>
+		<key>MUNKI_CATEGORY</key>
+		<string>Science</string>
+		<key>MUNKI_DEVELOPER</key>
+		<string>Continuum</string>
+		<key>MUNKI_REPO_SUBDIR</key>
+		<string>apps/%NAME%</string>
+		<key>pkginfo</key>
+		<dict>
+			<key>catalogs</key>
+			<array>
+				<string>%MUNKI_CATALOGS%</string>
+			</array>
+			<key>category</key>
+			<string>%MUNKI_CATEGORY%</string>
+			<key>description</key>
+			<string>Anaconda is the leading open data science platform powered by Python. The open source version of Anaconda is a high performance distribution of Python and R and includes over 100 of the most popular Python, R and Scala packages for data science.
+For information about working with Anaconda have a look at the following documentation: &lt;a href="https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html" target="_blank"&gt;Anaconda3 Documentation&lt;/a&gt;</string>
+			<key>developer</key>
+			<string>%MUNKI_DEVELOPER%</string>
+			<key>display_name</key>
+			<string>%NAME%3</string>
+			<key>name</key>
+			<string>%NAME%3</string>
+			<key>requires</key>
+			<array>
+				<string>python3</string>
```

Comment:
At the moment I added the requirement of installing python3, but I am not sure if we do want that because it is not the case for the previous anaconda releases.